src/main/resources/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.ui.xml

```diff
@@ -5,12 +5,13 @@
 	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html"
 	xmlns:w="urn:import:org.sagebionetworks.web.client.widget">
 	<g:HTMLPanel>
-		<b:Row>

```

Comment:
Rework the layout.  Instead of using the Bootstrap responsive Rows/Columns, add class names and define a css grid (that changes based on the screen size).